certbot/lock.py

```diff
@@ -208,26 +209,30 @@ class _WindowsLockMechanism(_BaseLockMechanism):
     A Windows lock file mechanism.
     By default on Windows, acquiring a file handler gives exclusive access to the process
     and results in an effective lock. However, it is possible to explicitly acquire the
-    file handler in shared access in terms of read and write, and this is done by os.open
+    file handler in shared access in terms of read and write, and this is done by filesystem.open
```

Comment:
```suggestion
    file handler in shared access in terms of read and write, and this is done by os.open
```

Looks like this was probably meant to be the original `os`

Reply:
Indeed, this is pycharm that replace this in comments during a safe replace operation.